include/targets/cuda/math_helper.cuh

```diff
@@ -2,6 +2,7 @@
 
 #include <cmath>
 #include <target_device.h>
+#include <quda_cuda_api.h>
```

Comment:
Why did this header need to be added here?

Reply:
Not sure why, but there probably was a reason.  I've reverted for now and will revisit if I encounter it again.
21d2cadf0cd2a4c23e4131b6bad0a3e92c9f7dc3